editoast/src/views/electrical_profiles.rs

```diff
@@ -120,7 +120,7 @@ impl ElectricalProfileSet {
 #[derive(Debug, Error, EditoastError)]
 #[editoast_error(base_id = "electrical_profiles", context = "Self::context")]
 pub enum ElectricalProfilesError {
-    /// Couldn't found the infra with the given id
+    /// Couldn't found the electrical profile with the given id
```

Comment:
Might as well :grin: 
```suggestion
    /// Couldn't find the electrical profile set with the given id
```